core/src/services/sqlite/backend.rs

```diff
@@ -188,7 +195,35 @@ impl Adapter {
     }
 }
 
+#[self_referencing]
+pub struct SqlStream {
```

Comment:
How about following the same name pattern like `SqliteScanner`

Reply:
Renamed.